sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -506,12 +508,16 @@ def __init__(self, options, root_staging_location=None):
         get_credentials=(not self.google_cloud_options.no_auth),
         http=http_client,
         response_encoding=get_response_encoding())
-    self._storage_client = storage.StorageV1(
-        url='https://www.googleapis.com/storage/v1',
-        credentials=credentials,
-        get_credentials=(not self.google_cloud_options.no_auth),
-        http=http_client,
-        response_encoding=get_response_encoding())
+    if storage_credentials:
```

Comment:
can we somehow add some comments to capture the discussions about the project option

Reply:
Done.